docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -579,6 +579,26 @@ Example:
     request_handle:setUpstreamOverrideHost("192.168.21.13", true)
   end
 
+.. _config_http_filters_lua_stream_handle_api_clear_route_cache:
+
+``clearRouteCache()``
+^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: lua
+
+  handle:clearRouteCache()
```

Comment:
Should this be named `recomputeRoute()` or `forceRecomputeRoute()` or something?

Reply:
`clear route cache` is used by both golang, dynamic modules, and wasm, I think keep consistant with them would be better. 🤔

Reply:
That's unfortunate, but makes sense